src/graph/session/GraphSessionManager.cpp

```diff
@@ -107,8 +113,16 @@ std::vector<meta::cpp2::Session> GraphSessionManager::getSessionFromLocalCache()
 
 folly::Future<StatusOr<std::shared_ptr<ClientSession>>> GraphSessionManager::createSession(
     const std::string userName, const std::string clientIp, folly::Executor* runner) {
+  // check the number of sessions per user per ip
+  auto key = userName + clientIp;
+  auto maxSessions = FLAGS_max_sessions_per_ip_per_user;
+  auto uiscFindPtr = userIpSessionCount_.find(key);
+  if (uiscFindPtr != userIpSessionCount_.end() && maxSessions > 0 && uiscFindPtr->second.get()->get() + 1 > maxSessions) {
```

Comment:
`uiscFindPtr->second.get()->get() + 1 > maxSessions` may cause overflow when the number of session reaches int32_max.
Use `uiscFindPtr->second.get()->get() > maxSessions -1` instead.

Also, please format this code block since it seems to exceed the maximum line width.